velox/common/caching/SsdFile.cpp

```diff
@@ -810,7 +811,9 @@ void SsdFile::initializeCheckpoint() {
   }
   bool hasCheckpoint = true;
   std::ifstream state(fileName_ + kCheckpointExtension);
-  if (!state.is_open()) {
+  if (state.is_open()) {
+    ++stats_.numCheckpoints;
```

Comment:
I think we shall have a counter inside readCheckpoint: numCheckpointReads?
And add a counter in SsdFile::checkpoint to record the sdd checkpoint write successes?